chcemvediet/apps/accounts/forms.py

```diff
@@ -65,15 +68,74 @@ def signup(self, request, user):
 class SettingsForm(forms.Form):
 
     anonymize_inforequests = forms.BooleanField(
-        label=_(u'accounts:SettingsForm:anonymize_inforequests:label'),
-        required=False,
-    )
+            label=_(u'accounts:SettingsForm:anonymize_inforequests:label'),
+            required=False,
+            )
+
+    custom_anonymization = forms.BooleanField(
+            label=_(u'accounts:SettingsForm:custom_anonymization:label'),
+            required=False,
+            help_text=_(u'accounts:SettingsForm:custom_anonymization:help_text'),
```

Comment:
@martinmacko47 Bolo by podla mna fajn, keby `help_text` zmizol ak user zaklikne moznost, ze chce anonymizovat vlastne stringy. Tam by to trebalo riesit najlepsie cez `javasript`, alebo?

Reply:
@viliambalaz vid https://github.com/martinmacko47/chcemvediet/pull/263#discussion_r375446869